ports/gstplugin/Cargo.toml

```diff
@@ -31,9 +31,9 @@ libservo = {path = "../../components/servo"}
 servo-media = {git = "https://github.com/servo/media"}
 sparkle = "0.1"
 # NOTE: the sm-angle-default feature only enables angle on windows, not other platforms!
```

Comment:
Remove this note now that it no longer applies?

Reply:
Done.